src/hardware/StandardLED.cpp

```diff
@@ -7,18 +7,16 @@
 #include "StandardLED.h"
 #include "GPIOPin.h"
 
-StandardLED::StandardLED(GPIOPin& gpioInstance, int featureFlag) :
-    gpio(gpioInstance), enabled(featureFlag != 0), inverted(featureFlag == 2) {
+StandardLED::StandardLED(GPIOPin& gpioInstance, const bool inverted) :
+    gpio(gpioInstance), inverted(inverted) {
 }
 
 void StandardLED::setGPIOState(bool state) {
-    if (enabled) {
-        if (inverted) {
-            gpio.write(state ? LOW : HIGH); // Inverted logic
-        }
-        else {
-            gpio.write(state ? HIGH : LOW); // Normal logic
-        }
+    if (inverted) {
+        gpio.write(state ? LOW : HIGH); // Inverted logic
+    }
+    else {
+        gpio.write(state ? HIGH : LOW); // Normal logic
```

Comment:
This entire block should be equivalent to a XOR

```suggestion
        gpio.write(state != inverted ? HIGH : LOW);
```